db/schema.rb

```diff
@@ -585,8 +585,11 @@
     t.string "administrative_district_code"
     t.string "administrative_district_name"
     t.string "slug"
+    t.boolean "section_41_approved", default: false, null: false
```

Comment:
are these suppose to be here? as one would expect the migration for these to be present in this pr

Reply:
uh oh